src/modules/ims_icscf/doc/ims_icscf_admin.xml

```diff
@@ -111,6 +111,106 @@ modparam("ims_icscf","scscf_entry_expiry", 300)
       </example>
     </section>
 
+    <section>
+      <title><varname>db_url</varname> (string)</title>
+
+      <para>
+        &url; of the database to be used.
+      </para>
+
+      <para>
+          If the db_url string is empty, the default database URL
+          will be used.
+      </para>
+
+      <para>
+        <emphasis>
+            Default value is <quote>&defaultrodb;</quote>.
+        </emphasis>
+      </para>
+
+      <example>
+        <title>Set <varname>db_url</varname> parameter</title>
+        <programlisting format="linespecific">
+...
+modparam("ims_icscf", "db_url", "mysql://username:password@localhost/kamailio")
```

Comment:
You could use also here the definition "&exampledb;

Reply:
Thanks, I applied it.